ui/admin/README.md

```diff
@@ -93,8 +98,44 @@ of testing".  Use test coverage as a guide to help you identify untested
 high-value code.
 
 We rely on `ember-a11y-testing` to validate accessibility in acceptance tests.
-If you write acceptance tests, please ensure at least one validation per
-route using `await a11yAudit();`.
+Write acceptance tests like normal. Our a11y testing strategy will automatically
+run `a11yAudit()` after [specific helper actions](tests/test-helper.js#L26), like visit, click, and fillIn.
+Our a11y tests have their own pnpm commands:
+
+* `pnpm test-ally` runs a11y tests for light and dark mode
+* `pnpm test-ally:light` runs a11y tests just for light mode
+* `pnpm test-ally:dark` runs a11y tests just for dark mode
```

Comment:
```suggestion
* `pnpm test-a11y` runs a11y tests for light and dark mode
* `pnpm test-a11y:light` runs a11y tests just for light mode
* `pnpm test-a11y:dark` runs a11y tests just for dark mode
```

Did we want to use `a11y` or `ally` for the commands? I notice the docs and what's in the package json are different. This also applies to the desktop ones

Reply:
Thanks for catching that. I will get those updated.